crates/config/src/lib.rs

```diff
@@ -2075,11 +2075,11 @@ impl Default for Config {
             prompt_timeout: 120,
             sender: Self::DEFAULT_SENDER,
             tx_origin: Self::DEFAULT_SENDER,
-            initial_balance: U256::from(0xffffffffffffffffffffffffu128),
+            initial_balance: U256::from((1u128 << 96) - 1),
             block_number: 1,
             fork_block_number: None,
             chain: None,
-            gas_limit: i64::MAX.into(),
+            gas_limit: (1u64 << 30).into(),
```

Comment:
nit: could we add the values as a comment?

Reply:
It's also present in config readme